From 629c8e10c88e3846c07a0d9f47f8fee20ebf3adb Mon Sep 17 00:00:00 2001
From: Nick Matantsev <nick.matantsev@gmail.com>
Date: Thu, 26 Aug 2021 00:46:08 -0400
Subject: [PATCH 30/31] Re-enable non-driver-model MMC init

---
 arch/arm/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/Kconfig b/arch/arm/Kconfig
index fbe90875ae..dc683a0b60 100644
--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -1025,7 +1025,7 @@ config ARCH_SUNXI
 	select DM_ETH
 	select DM_GPIO
 	select DM_KEYBOARD
-	select DM_MMC if MMC
+	select DM_MMC if MMC && !MACH_SUNIV # Lichee Pi Nano does not have driver-model MMC code
 	select DM_SCSI if SCSI
 	select DM_SERIAL
 	select DM_USB if DISTRO_DEFAULTS
-- 
2.41.0

